src/assets/Articles.jsx

```diff
@@ -1,10 +1,11 @@
 import { useEffect, useState } from "react";
 import axios from "axios";
 import { Link } from "react-router-dom";
+import "./CSS/Articles.css"
 
 export default function Articles() {
-  const [articles, setArticles] = uesState([]);
-  const [isLoading, setIsLoading] = useState(false);
+  const [Articles, setArticles] = useState([]);
```

Comment:
use articles not Articles here, shouldn't be capitalised. We do capitalise components but not state as a general rule